src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -90,4 +90,13 @@ public static boolean checkIsNotEqual(String string1, String string2, String err
         }
         return true;
     }
+
+    public static boolean checkIsEqual(String string1, String string2, String errorMsg, String qualifiedFieldName, Errors errors, String loggingContext) {
```

Comment:
I would have used the `StringValidators.checkIsNotEqual` and obviously negated the result on the `DueDiligenceValidator`  method for the `diligence`. Otherwise re-use it, something like this probably will do
```
public static boolean checkIsEqual(String string1, String string2, String errorMsg, String qualifiedFieldName, Errors errors, String loggingContext) {
    return !StringValidators.checkIsNotEqual(string1, string2, errorMsg, qualifiedFieldName, errors, loggingContext);
}
```

Reply:
Nope. Leave as you have done 👍